library/core/registry/src/main/java/org/quiltmc/qsl/registry/mixin/client/ItemRendererAccessor.java

```diff
@@ -0,0 +1,21 @@
+package org.quiltmc.qsl.registry.mixin.client;// Created 2022-04-12T11:27:59
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.client.color.item.ItemColors;
+import net.minecraft.client.render.item.ItemRenderer;
+
+/**
+ * Allows for ItemColors to be resynchronised with the registry, fixing the bug
+ * of the incorrect color renderer being applied to items due to the backing IDs
+ * being shifted.
+ *
+ * @author KJP12
+ * @since 3.0.0
+ **/
+@Mixin(ItemRenderer.class)
+public interface ItemRendererAccessor {
+	@Accessor("colors")
+	ItemColors getItemColors();
```

Comment:
Why isn't this called `getColors` to respect the accessor conventions?

Reply:
Fair, I'll go ahead and change it to be just `getColors`.